Lib/difflib.py

```diff
@@ -1187,7 +1187,11 @@ def unified_diff(a, b, fromfile='', tofile='', fromfiledate='',
         first, last = group[0], group[-1]
         file1_range = _format_range_unified(first[1], last[2])
         file2_range = _format_range_unified(first[3], last[4])
-        yield '@@ -{} +{} @@{}'.format(file1_range, file2_range, lineterm)
+        if isinstance(a, list) or isinstance(a, tuple) and \
```

Comment:
if isinstance(a, (list, tuple)) and isinstance(b, (list, tuple)):

Reply:
Thanks!